### PR TITLE
Fixed documentation typos

### PR DESCRIPTION
**Pull request by daphnied**
Fixes issue #[194]. 

Changes: [Fixed CONTRIBUTING.md typos.]

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,17 +1,16 @@
 # Contribute
 
 ### Step 1
--  Fork the repository
+-  Fork the repository.
```
Comment (jenkoian) [inline CONTRIBUTING.md]: If we're going to do this, should probably do it for all steps, personally don't think we need the full stop.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,17 +1,16 @@
...
 
 ### Step 2
 - Create a feature branch (git checkout -b your-new-feature)
 
 ### Step 3
-- Commit the changes (git commit -am 'Add some feature') (will additional add the new changes)
+- Commit the changes (git commit -m 'Add some feature') (will add the new changes)
```
Comment (jenkoian) [inline CONTRIBUTING.md]: I think the old one is fine, but additional should be additionally.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,17 +1,16 @@
...
 
 ### Step 4
 - Push to the branch (git push origin my-new-feature)
 
 ### Step 5
 - Create pull request
-
```
Comment (jenkoian) [inline CONTRIBUTING.md]: Can we keep the newline

Comment (daphnied) [reply CONTRIBUTING.md]: Yes that is fine

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -7,11 +7,10 @@
 - Create a feature branch (git checkout -b your-new-feature)
 
 ### Step 3
-- Commit the changes (git commit -am 'Add some feature') (will additional add the new changes)
+- Commit the changes (git commit -m 'Add some feature') (will additionally add the new changes)
```
Comment (jenkoian) [inline CONTRIBUTING.md]: Need the `a` too otherwise changes won't be added (as the bit in brackets suggests).

**Pull request closed without merging**